build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.MacOS.cs

```diff
@@ -21,5 +21,10 @@ partial class Paths
 			public static readonly string UnstrippedLibMonoSgenName = $"{LibMonoSgenBaseName}.d{Defaults.NativeLibraryExtension}";
 			public static readonly string StrippedLibMonoSgenName = $"{LibMonoSgenBaseName}{Defaults.NativeLibraryExtension}";
 		}
+
+		partial class Urls
+		{
+			public static readonly Uri DotNetInstallScript = new Uri ("https://dot.net/v1/dotnet-install.sh");
```

Comment:
Since URLs for `macOS` and `Linux` are the same, this partial class should be added to `Configurables.Unix.cs` instead